glib/src/variant.rs

```diff
@@ -1757,6 +1757,188 @@ impl<A: AsRef<[T]>, T: FixedSizeVariantType> ToVariant for FixedSizeVariantArray
     }
 }
 
+/// A wrapper type around `Variant` handles.
+#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct Handle(pub i32);
```

Comment:
Any reason why the inner value is public?

Reply:
Just because "why not" :) There's no problem giving direct access to it

Reply:
I guess I have no idea in which context you would use this. Making it public allows people to construct a new `Handle` that might contain a garbage value but then we also have a From implementation so :)

Reply:
It's a plain integer that is an index into an out of band transmitted array (of which you know the size). No problem here

Reply:
Also you can easily parse any kind of integer as a handle from a string anyway.